sim/hunter/arcane_shot.go

```diff
@@ -0,0 +1,46 @@
+package hunter
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (hunter *Hunter) registerArcaneShotSpell() {
+
+	dmgMultiplier := 1.0
+	if hunter.HasPrimeGlyph(proto.HunterPrimeGlyph_GlyphOfArcaneShot) {
+		dmgMultiplier = 1.12
+	}
+	hunter.ArcaneShot = hunter.RegisterSpell(core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: 3044},
+		SpellSchool: core.SpellSchoolArcane,
+		ProcMask:    core.ProcMaskRangedSpecial,
+		Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagAPL,
+
+		FocusCost: core.FocusCostOptions{
+			Cost: 25 - float64(hunter.Talents.Efficiency),
+		},
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: time.Second,
+			},
+			IgnoreHaste: true,
+		},
+
+		BonusCritRating: 0,
+		DamageMultiplierAdditive: 1,
+		DamageMultiplier: dmgMultiplier,
+		CritMultiplier: hunter.CritMultiplier(true, true, false),
+		ThreatMultiplier: 1,
+
+		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+			wepDmg := hunter.AutoAttacks.Ranged().CalculateWeaponDamage(sim, spell.RangedAttackPower(target))
```

Comment:
Not normalized? Just asking to make sure

Reply:
https://www.wowhead.com/cata/spell=3044/arcane-shot Little bit unclear here. The modifier seems to be not normalized, but effect 1 is. Idk which to use here, gotta revisit it

Reply:
Seems it should be normalized